code/game/gamemodes/dynamic/dynamic_simulations.dm

```diff
@@ -119,19 +119,5 @@
 
 	return export
 
-/client/proc/export_dynamic_json()
```

Comment:
Is this being removed because it's no longer functional?

Reply:
i think so

Reply:
roundstart_rules won't work for sure